docs/fundamentals/executor/hub/use-hub-executor.md

```diff
@@ -153,3 +153,11 @@ jina hub pull jinahub://<UUID>[:<SECRET>][/<TAG>]
 
 
 The source code of the Executor will be stored at `~/.jina/hub-packages`.
+
+### list the local location of executors
```

Comment:
```suggestion
### List locations of local Executors
```

Reply:
Thanks, i will convert  `list the local location of executors`  to` List locations of local Executors`